lib/bolt/cli/src/commands/admin/cluster/datacenter.rs

```diff
@@ -0,0 +1,92 @@
+use anyhow::*;
+use bolt_core::{
+	context::ProjectContext,
+	tasks::{self, api::send_request},
+	utils,
+};
+use clap::Parser;
+use rivet_api::{
+	apis::{configuration::Configuration, *},
+	models,
+};
+use uuid::Uuid;
+
+#[derive(Parser)]
+pub enum SubCommand {
+	/// Creates a new datacenter
+	Create {
+		/// The ID of the cluster that the datacenter will belong to
+		#[clap(short, long)]
+		cluster_id: String,
+	},
+	/// Deletes an existing datacenter
+	Delete {
+		/// The ID of the cluster that the datacenter belongs to
+		#[clap(short, long)]
+		cluster_id: String,
```

Comment:
The cluster should be inherited to the dc commands via a global argument, so the command syntax follows `bolt admin cluster <cluster_name_id> dc <dc_name_id>`